llvm/docs/FatLTO.rst

```diff
@@ -76,3 +76,40 @@ Supported File Formats
 The current implementation only supports ELF files. At time of writing, it is
 unclear if it will be useful to support other object file formats like ``COFF``
 or ``Mach-O``.
+
+Usage
+===========
+
+Users are expected to pass ``-ffat-lto-objects`` to clang in addition to one of
```

Comment:
Clang users can specify `-ffat-lto-objects` along with `-flto` or `-flto=thin`. Without the ``-flto`` flag, ``-ffat-lto-objects`` has no effect.

Reply:
done